spark-connector/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+plugins {
+  `maven-publish`
+  id("java")
+  id("idea")
+}
+
+repositories {
+  mavenCentral()
+}
+
+val scalaVersion: String = project.properties["scalaVersion"] as? String ?: extra["defaultScalaVersion"].toString()
+val sparkVersion: String = libs.versions.spark.get()
+val icebergVersion: String = libs.versions.iceberg.get()
+val kyuubiVersion: String = libs.versions.kyuubi.get()
+
+dependencies {
+  implementation(project(":clients:client-java"))
```

Comment:
You should depend on client java runtime, not client java

Reply:
could you detail the reason?

Reply:
client java runtime shades the dependencies without conflict, trino also use that one. See trino's implementation.

Reply:
done